libpoolprotocols/stratum/EthStratumClient.cpp

```diff
@@ -56,6 +56,7 @@ EthStratumClient::EthStratumClient(Farm* f, MinerType m, string const & host, st
 
 EthStratumClient::~EthStratumClient()
 {
+	m_running = false;
```

Comment:
This variable serves no viable purpose! It does not belong in the stratum client. If a client is instantiated, then it is by definition 'running'.